packages/salesforcedx-vscode-apex/src/messages/i18n.ts

```diff
@@ -24,6 +24,11 @@ export const messages = {
   source_missing_text: '%s points to a missing folder',
   java_runtime_missing_text:
     'Java runtime could not be located. Set one using the salesforcedx-vscode-apex.java.home VS Code setting.',
+  force_apex_test_run_codeAction_description_text: 'Run Apex test(s)',
+  force_test_view_loading_message: 'Loading Apex Tests ...',
```

Comment:
We usually use sentence-case capitalization for progress messages: "Loading Apex tests..."